validator/src/main.rs

```diff
@@ -1331,6 +1331,15 @@ pub fn main() {
                 .help("Gossip DNS name or IP address for the validator to advertise in gossip \
                        [default: ask --entrypoint, or 127.0.0.1 when --entrypoint is not provided]"),
 
+        )
+        .arg(
+            Arg::with_name("overwrite_tpu_addr")
+                .long("overwrite-tpu-addr")
+                .value_name("OVERWRITE_TPU_ADDR")
+                .takes_value(true)
+                .validator(solana_net_utils::is_host_port)
+                .help("Force overwrite the tpu address to advertise in gossip"),
```

Comment:
This can become...

```suggestion
                .help("Specify TPU bind address to [default: see --bind-address]"),
```

Reply:
it's not to bind, it's what to advertise in gossip

Reply:
`"Specify TPU address advertised in gossip[default: see --bind-address]"`?

Reply:
""Specify TPU address advertised in gossip[default: see --bind-address]"?"

This one sounds fine to me.

Reply:
> it's not to bind, it's what to advertise in gossip

what good is advertising without binding?

Reply:
Sending TPU traffic to a proxy host/service, which then forwards it to the "real" bind address